scripts/submission/submit_to_gencc.py

```diff
@@ -0,0 +1,496 @@
+    """    
+    if path:
+        gencc_dir = path + create_datetime_now()
+        output_file = f"{gencc_dir}/G2P_GenCC.txt"
+        final_output_file = f"{gencc_dir}/G2P_GenCC.xlsx"
+    else:
+        output_file = "G2P_GenCC.txt"
+        final_output_file = "G2P_GenCC.xlsx"
+    return gencc_dir if path else ".", output_file, final_output_file
+
+def handle_new_submission(read_data: list, output_file: str, dry: bool, db_config: dict[str, Any]) -> write_to_the_GenCC_file:
+    """Handles new submission, so new generation of the G2P records and GenCC data 
+
+    Args:
+        read_data (list): Data from the G2P all records file
+        output_file (str): Output text file where the submission will be written into
+        dry (bool): Boolean (args.dry)
+        db_config (dict[str, Any]): DB/API configuration
+
+    Returns:
+        write_to_the_GenCC_file: A text file containing all the records to be submitted to GenCC
+    """    
+    return write_to_the_GenCC_file(read_data, output_file, str(dry), db_config)
+
+def handle_existing_submission(read_data: list, output_file: str, dry: bool, db_config: dict[str, Any], old_file: str) -> write_to_the_GenCC_file:
+    """Handles existing submission.
+
+    Args:
+        read_data (list): Data from the G2P all records file
+        output_file (str): Output text file where the submission will be written into
+        dry (bool): Boolean (args.dry)
+        db_config (dict[str, Any]): DB/API configuration
+        old_file (str) : Old file containing submitted GenCC submission to allow comparison
+
+    Returns:
+        write_to_the_GenCC_file: A text file containing all the records to be submitted to GenCC
+    """    
+    unsubmitted = get_unsubmitted_record(db_config)
+    common = retrieve_unsubmitted_records(read_data, unsubmitted)
+
+    if old_file:
+        old_reader = read_from_old_gencc_submission(old_file)
+        later_review = get_later_review_date(db_config)
+        compared = compare_data_changes(old_reader, later_review, read_data, db_config)
+        merged_data = add_unsubmitted_ids_and_later_review_date(compared, common)
+        return write_to_the_GenCC_file(merged_data, output_file, str(dry), db_config)
+    else:
+        return write_to_the_GenCC_file(common, output_file, str(dry), db_config)
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument(
+        "-p",
+        "--path",
+        help="Path where the G2P and GenCC files are going to be saved",
+        required=False,
+    )
+    ap.add_argument("--config_file", required=True, help="G2P Configuration file")
+    ap.add_argument(
+        "--write_to_db",
+        required=False,
+        action="store_true",
+    )
+    ap.add_argument(
+        "--new",
+        action="store_true",
+        required=False,
+        help="If new submission, true then it requires a whole new submission and does no checks",
```

Comment:
This sentence is confusing, I can't understand what the option does when I read the description.